src/Raven.Server/RavenServer.cs

```diff
@@ -1615,6 +1616,23 @@ public AuthenticationStatus Status
                 set => _status = value;
             }
 
+            public AuthorizationStatus GetAuthorizationStatus(string databaseName)
+            {
+                switch (Status)
+                {
+                    case AuthenticationStatus.ClusterAdmin:
+                        return AuthorizationStatus.ClusterAdmin;
+                    case AuthenticationStatus.Operator:
+                        return AuthorizationStatus.Operator;
+                    case AuthenticationStatus.Allowed:
+                        if (AuthorizedDatabases.TryGetValue(databaseName, out var databaseAccess) && (databaseAccess == DatabaseAccess.Admin))
+                            return AuthorizationStatus.DatabaseAdmin;
+                        return AuthorizationStatus.ValidUser;
```

Comment:
What if `databaseAccess` value is `DatabaseAccess.Read`? How do we ensure that a read only certificate won't be enough do to the export of data?

Reply:
If `DatabaseAccess.Read` is not possible then handle that (throw?) explicitly here I think.

Reply:
DatabaseAccess.Read is not possible for now. I added a case for read

Reply:
If `DatabaseAccess.Read` is not possible here then why the following code was added:

```
if (databaseAccess == DatabaseAccess.Read)
     return AuthorizationStatus.RestrictedAccess;
```
?

Reply:
This is just a precaution in case any endpoint will use this and will have `Read` access.
Currently, the import endpoints are `ReadWrite` or `Admin`.

Reply:
OK